exopy/tasks/tasks/logic/loop_task.py

```diff
@@ -234,3 +234,4 @@ def _post_setattr_name(self, old, new):
         """
         if self.task:
             self.task.name = new
+        super(LoopTask, self)._post_setattr_name(old, new)
```

Comment:
We can use the argument less super here since we do not support Python 2 anymore.